### ISSUE TITLE
How to change the color?

### ISSUE THREAD

**Issue by HanielF** [perryrh0dan/taskline#10]
## Describe:
Because the background color is similar to the taskline color, the id of tasks and notes cannot be distinguished clearly.

## Question:
How to change the color of id numbers?

## Screenshot:
![image](https://user-images.githubusercontent.com/24732516/66905160-e9ecf980-f037-11e9-8948-c59acb585628.png)

**Comment by perryrh0dan** [comment]
Hi HanielF, 

have a look at the [readme](https://github.com/perryrh0dan/taskline#themes). 
If this is not working or you have some problems feel free to contact me.

BR.
Thomas

**Comment by HanielF** [comment]
Thanks for your reply!

I have seen readme in both dev and master branch before. I tried to modify `~/.taskline.json`, but it doesn't work.

Even though i changed the color of pale, note and star to red, it still looks like the screenshot below.
![image](https://user-images.githubusercontent.com/24732516/66908326-74842780-f03d-11e9-9741-eb50147701f2.png)

Part of my `~/.taskline.json`:
```
  "theme": {
    "colors": {
      "pale": "red",
      "task": {
        "priority": {
          "medium": "yellow",
          "high": "red"
        }
      },
      "icons": {
        "note": "red",
        "success": "green",
        "star": "red",
        "progress": "blue",
        "pending": "magenta",
        "canceled": "red"
      }
    }
  }
}
```

**Comment by perryrh0dan** [comment]
Are you using the snap version or the npm version ?

**Comment by HanielF** [comment]
Snap used.
![image](https://user-images.githubusercontent.com/24732516/66913193-be254000-f046-11e9-8894-8b0278731610.png)

What's more, the system is archlinux and the desktop environment is Gnome.

**Comment by perryrh0dan** [comment]
Mhh.. on my computer it just works fine. Are you using the taskline.json file in the snap directory ? Something like `/home/<username>/snap/taskline/current/.taskline.json`

**Comment by perryrh0dan** [comment]
Maybe it would be also helpfull to have a command like `taskline config` to display the current active config ?

**Comment by HanielF** [comment]
It works! Thank you very much!

**Comment by aditya95sriram** [comment]
Yeah, this same issue tripped me as well. I created/edited the `~/.taskline.json` file and tried a bunch of things like restarting the terminal, trying with other shells etc. but nothing seemed to affect the colors. Glad I found this thread. @perryrh0dan , yes adding a `taskline config` seems like one of those things that's not too difficult to implement but would prove immensely helpful, maybe it can even print out the path of the config file along with the actual config options.
